### PR TITLE
Bookstore setup

### PR DESCRIPTION
**Pull request by Outkarm**
in this project, the basic UI for a Bookstore app was built using react

### CODE REVIEW

File: src/components/Authors.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
```
Comment (AmaduKamara) [inline src/components/Authors.js]: - [ ] Let's try not to disable Linter checks. Linter checks help us write clean, better, and professional code and help us become better developers. Disabling Linters defies its purpose and hence, can't help you become a better and professional developer. Please remove the ESLint disable and try to fix the error 👍🏽

File: src/components/Bookstore.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  ];
+
+  return (
+    <div>
+      <ul className="the-books">
+        {Books.map((book) => (
+          <li className="container" key={book.id}>
+            <BookTitle title={book.title} author={book.author} />
+            <button type="submit">Remove</button>
+          </li>
```
Comment (AmaduKamara) [inline src/components/Bookstore.js]: - [ ] The remove button is supposed to be inside the single book component. In your case, that's the BookTitle component inside the Authors.js file. Kindly move this button into that component. 👍🏽

[ Optional Change ]
- [ ] It is s good convention to name your files and your components the same name. For instance, your BookTitle component could better be named Book, and the file name also is better named Book.js. This makes your code better organized and easy to read, find and solve bugs.  👍🏽

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
+            </Link>
+          </li>
+          <li>
+            {' '}
+            <Link className="category" to="/Category">
+              Category
+            </Link>
+          </li>
+        </ul>
+      </nav>
```
Comment (AmaduKamara) [inline src/App.js:26]: [ Optional Changes ]

The beauty of React is the component-based architecture. You can build your whole app in conjunction with tiny tiny components.

- [ ] What if you could move this nav code into a separate Navbar.js component and then import the Navbar component here? I think that will make your App.js component lean and clean. Please try it and see how clean your code will be 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Outkarm,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ You used create-react-app for project initialization 👍🏽
✅ Used React router dom well 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on fixing the Linter errors so you have a professional codebase 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**